README.md

```diff
@@ -37,6 +37,71 @@ Latest API Docs is <a target="_blank" href="https://wso2-extensions.github.io/si
      </dependency>
 ```
 
+## Running Integration tests in docker containers(Optional)
+
+1. Install and run docker in daemon mode.
+
+    *  How to install docker on Linux
+    ```    
+    wget -qO- https://get.docker.com/ | sh
+    sudo service dockerd stop
+    export DOCKER_HOST=tcp://172.17.0.1:4326
+    docker daemon -H tcp://172.17.0.1:4326
+    ```
+    *  How to install docker on Mac 
+   https://docs.docker.com/docker-for-mac/
+
+    *  How to install docker on Windows
+   https://docs.docker.com/docker-for-windows/
+2. Go inside to the siddhi-store-rdbms directory
```

Comment:
The 2nd and 3rd steps should be combined. Also, give the complete path to the relevant directory instead of specifying just the directory name.

Suggestion:
"To run the integration test, navigate to the `<PATH>' and issue the following commands."